FirebaseInstallations/Source/Tests/Unit/FIRInstallationsAPIServiceTests.m

```diff
@@ -83,8 +83,7 @@ - (void)testRegisterInstallationSuccess {
     XCTAssertEqualObjects(body[@"authVersion"], @"FIS_v2");
     XCTAssertEqualObjects(body[@"appId"], installation.appID);
 
-    // TODO: Find out what the version should we pass and test.
-    //    XCTAssertEqualObjects(body[@"sdkVersion"], @"a1.0");
+    XCTAssertEqualObjects(body[@"sdkVersion"], @"0.1.0");
```

Comment:
Hmm, this will be painful updating each time. Can we compare it against the constant instead?

Reply:
Updated. I had to use the same code to calculate the version in tests, so now the tests validate less - only the fact the header was added, which looks fine. Let me know if you have a better idea. Thanks.